Source/Moq.Analyzers/MoqMethodDescriptor.cs

```diff
@@ -1,14 +1,15 @@
-﻿using System.Text.RegularExpressions;
+﻿using System.Diagnostics;
+using System.Text.RegularExpressions;
 
 namespace Moq.Analyzers;
 
 internal class MoqMethodDescriptor
 {
-    private readonly bool isGeneric;
+    private readonly bool _isGeneric;
```

Comment:
@MattKotsenas There are some conflicting style rules with this. We can take it up as a separate issue and PR, this is big enough already